mfkdf2/src/setup/factors/mod.rs

```diff
@@ -1,87 +1,88 @@
+pub mod hmacsha1;
 pub mod hotp;
+pub mod ooba;
+pub mod passkey;
 pub mod password;
-// pub mod question;
-// pub mod stack;
-// pub mod totp;
-// pub mod uuid;
+pub mod question;
+pub mod stack;
+pub mod totp;
+pub mod uuid;
 
-// pub use hmacsha1::hmacsha1;
+pub use hmacsha1::hmacsha1;
 pub use hotp::hotp;
+pub use passkey::passkey;
 pub use password::password;
-// pub use question::question;
-// pub use stack::stack;
-// pub use uuid::uuid;
+pub use question::question;
+pub use stack::stack;
+pub use totp::totp;
+pub use uuid::uuid;
 
 #[derive(Clone, Debug, Serialize, Deserialize, uniffi::Enum)]
-pub enum FactorType {
+pub enum FactorSetupType {
   Password(password::Password),
   HOTP(hotp::HOTP),
-  // Question(question::Question),
-  // UUID(uuid::UUID),
-  // HmacSha1(hmacsha1::HmacSha1),
-  // TOTP(totp::TOTP),
-  // Stack(stack::Stack),
+  Question(question::Question),
+  UUID(uuid::UUID),
+  HmacSha1(hmacsha1::HmacSha1),
+  TOTP(totp::TOTP),
+  OOBA(ooba::Ooba),
+  Passkey(passkey::Passkey),
+  Stack(stack::Stack),
 }
 
-impl FactorType {
-  pub fn inner(&self) -> &dyn FactorTrait {
+impl FactorSetupType {
+  pub fn inner(&self) -> &dyn FactorSetupTrait {
     match self {
-      FactorType::Password(password) => password,
-      FactorType::HOTP(hotp) => hotp,
-      // FactorType::Question(question) => question,
-      // FactorType::UUID(uuid) => uuid,
-      // FactorType::HmacSha1(hmacsha1) => hmacsha1,
-      // FactorType::TOTP(totp) => totp,
-      // FactorType::Stack(stack) => stack,
+      FactorSetupType::Password(password) => password,
+      FactorSetupType::HOTP(hotp) => hotp,
+      FactorSetupType::Question(question) => question,
+      FactorSetupType::UUID(uuid) => uuid,
+      FactorSetupType::HmacSha1(hmacsha1) => hmacsha1,
+      FactorSetupType::TOTP(totp) => totp,
+      FactorSetupType::OOBA(ooba) => ooba,
+      FactorSetupType::Passkey(passkey) => passkey,
+      FactorSetupType::Stack(stack) => stack,
     }
   }
 
-  pub fn inner_mut(&mut self) -> &mut dyn FactorTrait {
+  pub fn inner_mut(&mut self) -> &mut dyn FactorSetupTrait {
     match self {
-      FactorType::Password(password) => password,
-      FactorType::HOTP(hotp) => hotp,
-      // FactorType::Question(question) => question,
-      // FactorType::UUID(uuid) => uuid,
-      // FactorType::HmacSha1(hmacsha1) => hmacsha1,
-      // FactorType::TOTP(totp) => totp,
-      // FactorType::Stack(stack) => stack,
+      FactorSetupType::Password(password) => password,
+      FactorSetupType::HOTP(hotp) => hotp,
+      FactorSetupType::Question(question) => question,
+      FactorSetupType::UUID(uuid) => uuid,
+      FactorSetupType::HmacSha1(hmacsha1) => hmacsha1,
+      FactorSetupType::TOTP(totp) => totp,
+      FactorSetupType::OOBA(ooba) => ooba,
+      FactorSetupType::Passkey(passkey) => passkey,
+      FactorSetupType::Stack(stack) => stack,
     }
   }
 }
 
-impl FactorTrait for FactorType {
+impl FactorSetupTrait for FactorSetupType {
```

Comment:
Just call it `FactorSetup`, don't need to end with `..Trait`

I did that because I was just running into naming issues.